src/views/partials/email-preview.js

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+function getPreviewOptions (currentTemplateKey, data) {
+  const optionsElements = Object.keys(data).map(templateKey => (`
+    <option
+      value='${templateKey}'
+      ${currentTemplateKey === templateKey ? 'selected' : ''}
+    >
+      ${data[templateKey].label}
+    </option>
+  `)).join('')
+
+  return optionsElements
+}
+
+function getRecipientInputs (recipients) {
+  const recipientInputElements = recipients.map((recipient, index) => (`
+    <label>
+      <input
+        name="email-recipient-option"
+        type="radio"
+        value="${recipient}"
+        ${index === 0 ? 'checked' : ''}
+      >
+      ${recipient}
+    </label>
+  `)).join('')
+
+  return `
+    <fieldset>
+      ${recipientInputElements}
+    </fieldset>
+  `
+}
+
+export const emailPreview = data => {
+  const {
+    template: currentTemplateKey,
+    data: emailTemplates,
+    recipients
+  } = data.email
+  const selectedPreviewTemplate = emailTemplates[currentTemplateKey]?.template
+
+  return `
```

Comment:
Per comment above, could we simply add the static script tag here?  It's `type="module"` so it should still be deferred and loaded as expected?

```suggestion
  return `
  <script src="/js/partials/email-preview.js" type="module"></script>
```

UPDATE: suggested change works right now via `npm run dev`, but to work for `npm start` depends on this new code in the add-email PR:
https://github.com/mozilla/blurts-server/blob/11ec0ba60e03dabff53b64d089d7f932778540bc/src/esbuild.js#L18-L21

(which will be merged shortly...)

Reply:
Update, ☝️ this code was merged, so pulling `main` into your branch should enable the correct functionality if you decide to implement.

Reply:
Addressed in [8bd7bab](https://github.com/mozilla/blurts-server/pull/2790/commits/8bd7babdb6fd7baa1678823bd18d5a9cbc9f0145).